### PR TITLE
feat: upgrade allure report to ver 2.x.x

### PR DESCRIPTION
**Pull request by khiemkao**
upgrade Allure to 2.x.x version
clean up pom.xml
clean up azure-pipelines.yml

### CODE REVIEW

**Review by LuanNguyenBao — APPROVED** [review]
The PR looks good to me, please help us to inform in group chat for the replacement annotations of allure version 2.x. Thank anh.

File: pom.xml [hunk 1]
```diff
@@ -9,145 +9,86 @@
...
-
-        <!-- https://mvnrepository.com/artifact/io.github.bonigarcia/webdrivermanager -->
         <dependency>
             <groupId>io.github.bonigarcia</groupId>
             <artifactId>webdrivermanager</artifactId>
-            <version>5.3.0</version>
-        </dependency>
-
-        <!-- https://mvnrepository.com/artifact/org.slf4j/slf4j-api -->
-        <dependency>
```
Comment (tuanna0308) [inline pom.xml:78]: Hi anh Khiem, Do you really want to remove the "slf4j" library?

File: pom.xml [hunk 2]
```diff
@@ -9,145 +9,86 @@
     <version>1.0-SNAPSHOT</version>
 
     <properties>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <maven.compiler.source>8</maven.compiler.source>
-        <maven.compiler.target>8</maven.compiler.target>
-
+        <maven.conpiler.version>3.8.1</maven.conpiler.version>
```
Comment (tuanna0308) [inline pom.xml:12]: Hi anh Khiem, please correct the gramma in the "maven.conpiler.version", should change it to "maven.compiler.version"

File: pom.xml (hunk 1 above)
Comment (khiemkao) [reply pom.xml:78]: @tuanna0308 do we use this lib in the project? if yes, I will revert this

**Review by tuanna0308 — APPROVED** [review]
Hi anh Khiem, this PR looks good for me, I will merge this branch

**Pull request merged**